client/components/dot-pager/style.scss

```diff
@@ -74,8 +78,17 @@
 
 		&.dot-pager__control-prev,
 		&.dot-pager__control-next {
-			width: 18px;
-			height: 18px;
+			// Allow the control buttons the space they need for labels
+			width: auto;
+			height: auto;
+
+			// align label and icon
+			align-items: end;
```

Comment:
Is there any problem if we remove the above three?

Reply:
Nice! We don't actually need the align-items any more. We don't really need the height either.